### PR TITLE
Update ORA doc for course teams

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3491](https://openedx.atlassian.net/browse/DOC-3491)

Took a fresh look at the ORA doc for course teams - cleaned up and simplified the language where possible, updated screenshots to reflect current UI, removed unnecessary screenshots. As part of updates, also addressed DOC-713 (how to deal with ungraded responses = staff assessments) and DOC-3156 (ORA not supported as a prerequisite subsection).

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Product: @sstack22 
- [x] Doc team review: @edx/doc 

FYI: @jaakana, @mmacfarlane, @jhendersonedx @kathleenduffy

### HTML Version
- [x] http://draft-ca-ora-update.readthedocs.io/en/latest/exercises_tools/open_response_assessments/index.html

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst [hunk 1]
```diff
@@ -184,22 +197,18 @@ To add the rubric, follow these steps.
...
+To provide a comment field without options, complete these steps.
 
-.. image:: ../../../../shared/images/PA_0_Option_Criteria.png
-  :alt: Examples of criteria with and without a comment field.
+#. In the ORA component editor, select the **Rubric** tab.
 
-To provide a comment field without options, complete these steps.
+#. In the **Criterion** section for the criterion that you want to only
+   provide a comment field for, select **Remove** to remove each option.
+   the options.
```
Comment (grantgoodmanedX) [inline en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst]: Delete surplus "the options."

File: en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst [hunk 2]
```diff
@@ -210,32 +219,31 @@ Step 4. Specify the Assignment Name and Response Dates
...
+  such as `Time and Date Time Zone Converter
   <http://www.timeanddate.com/worldclock/converter.html>`_.
 
 To specify a name for the assignment as well as start and due dates for all
 learner responses, complete these steps.
 
-#. In the component editor, select the **Settings** tab.
+#. In the ORA component editor, select the **Settings** tab.
 
 #. Next to **Display Name**, type the name you want to give the assignment.
```
Comment (grantgoodmanedX) [inline en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst]: "enter the name you want to give"

File: en_us/shared/subsections/open_response_assessments/OpenResponseAssessments.rst [hunk 3]
```diff
@@ -511,20 +317,15 @@ do not count toward the score that the response receives.
...
-below the entire rubric. You can also add a comment field to an individual
-criterion or to several individual criteria. This comment field can contain up
-to 300 characters.
+By default, in peer assessment steps, learners can provide text feedback for the
+entire response, using a single comment field below the entire rubric. You can
+also add a comment field to an individual criterion or to several individual
+criteria. This comment field can contain up to 300 characters.
 
 The comment field appears below the options for the criterion. In the following
 image, both criteria have a comment field. There is also a field for overall
```
Comment (grantgoodmanedX) [inline en_us/shared/subsections/open_response_assessments/OpenResponseAssessments.rst]: The following image has been removed, so you should re-write this paragraph or restore the image.

**Review by grantgoodmanedX — COMMENTED** [review]
Looks good, with just a few comments. +1

File: en_us/shared/subsections/open_response_assessments/OpenResponseAssessments.rst (hunk 3 above)
Comment (catong) [reply en_us/shared/subsections/open_response_assessments/OpenResponseAssessments.rst]: Oops! Thanks for catching that.

File: en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst (hunk 2 above)
Comment (catong) [reply en_us/shared/subsections/open_response_assessments/CreateORAAssignment.rst]: Aha!

**Comment by catong** [comment]
@sstack22 Do you want to take a look before I merge this PR?

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1421

**Pull request merged**